api/src/parcel/get.handler.ts

```diff
@@ -0,0 +1,96 @@
+import { APIGatewayProxyResult } from 'aws-lambda';
+import * as AWS from 'aws-sdk';
+import { RDSDataService } from 'aws-sdk';
+import { ExecuteStatementRequest, SqlParametersList } from 'aws-sdk/clients/rdsdataservice';
+
+const CORS_HEADERS = {
+  'Access-Control-Allow-Headers': 'Content-Type',
+  'Access-Control-Allow-Origin': '*',
+  'Access-Control-Allow-Methods': 'OPTIONS,GET',
+};
+const SCHEMA = 'public';
+const SQL_QUERY = `SELECT address,
+                          public_lead_prediction,
```

Comment:
should these be low/high estimates?

Reply:
Added the prediction lambda to this PR to read in the schema values